rskj-core/src/test/java/co/rsk/peg/feeperkb/FeePerKbSupportImplTest.java

```diff
@@ -150,6 +150,48 @@ void voteFeePerKbChange_successfulFeePerKbChanged_shouldReturnSuccessfulFeeVoted
         assertEquals(expectedResult, actualResult);
     }
 
+    @Test
+    void voteFeePerKbChange_zeroFeePerKbValue_shouldReturnUnsuccessfulFeeVotedResponseCode() {
+        SignatureCache signatureCache = mock(SignatureCache.class);
+        Transaction tx = getTransactionFromAuthorizedCaller(signatureCache);
+        Coin zeroFeePerKb = Coin.ZERO;
+
+        Integer actualResult = feePerKbSupport.voteFeePerKbChange(tx, zeroFeePerKb, signatureCache);
+
+        Integer expectedResult = FeePerKbResponseCode.NEGATIVE_FEE_VOTED.getCode();
+        assertEquals(expectedResult, actualResult);
+    }
+
+    @Test
+    void voteFeePerKbChange_veryLowFeePerKbValue_shouldReturnUnsuccessfulFeeVotedResponseCode() {
```

Comment:
The name needs an update now, no longer returns unsuccessful

Reply:
Done!